pgcli/completion_refresher.py

```diff
@@ -140,3 +136,7 @@ def refresh_casing(completer, executor):
     if os.path.isfile(casing_file):
         with open(casing_file, 'r') as f:
             completer.extend_casing([line.strip() for line in f])
+
+@refresher('functions')
```

Comment:
`PGCompleter.extend_functions()` uses the casing info (in `_format_arg()`) added by `PGCompleter.extend_casing()` (and the refresher functions are called in the order they're defined) so the casing and function refreshers need to be in that order.